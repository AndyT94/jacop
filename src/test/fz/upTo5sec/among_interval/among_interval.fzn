var 3..3: low:: output_var;
var 5..5: up:: output_var;
array [1..5] of var int: x:: output_array([1..5]) = [4,5,8,4,1];
solve  satisfy;
