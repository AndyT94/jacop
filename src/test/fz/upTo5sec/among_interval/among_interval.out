low = 3;
up = 5;
x = array1d(1..5, [4, 5, 8, 4, 1]);
----------